.github/workflows/plugin_upstream_testing_base.yml

```diff
@@ -20,13 +20,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        nautobot-version: ["next", "develop"]
+        nautobot-version: ["develop", "ltm-1.6"]
```

Comment:
This is used to construct the Docker image tag, since we can't use a `/` in the tag name.